--- build_tools/menuselect-deps.in	(Asterisk 13.13.1)
+++ build_tools/menuselect-deps.in	(working copy)
@@ -9,2 +9,3 @@
 DLADDR=@PBX_DLADDR@
+EVS=@PBX_EVS@
 FLEX=@PBX_FLEX@
--- configure.ac	(Asterisk 13.13.1)
+++ configure.ac	(working copy)
@@ -460,2 +460,3 @@
 AST_EXT_LIB_SETUP([DAHDI], [DAHDI], [dahdi])
+AST_EXT_LIB_SETUP([EVS], [3GPP EVS Audio Decoder/Encoder], [3gpp-evs])
 AST_EXT_LIB_SETUP([FFMPEG], [Ffmpeg and avcodec], [avcodec])
@@ -2356,2 +2357,4 @@
 
+AST_EXT_LIB_CHECK([EVS], [3gpp-evs], [evs_enc], [3gpp-evs/prot.h])
+
 AST_EXT_LIB_CHECK([SPEEX], [speex], [speex_encode], [speex/speex.h], [-lm])
--- makeopts.in	(Asterisk 13.13.1)
+++ makeopts.in	(working copy)
@@ -135,2 +135,5 @@
 
+EVS_INCLUDE=@EVS_INCLUDE@
+EVS_LIB=@EVS_LIB@
+
 CURL_INCLUDE=@CURL_INCLUDE@
